ide/Espressif/ESP-IDF/component-manager/CMakeLists.txt

```diff
@@ -0,0 +1,111 @@
+#
+# Copyright (C) 2006-2022 wolfSSL Inc.  All rights reserved.
```

Comment:
It's 2023.

Reply:
And this is the commercial licensing header. Everything in wolfSSH must be GPLv3.